src/handlers/publicResolver.ts

```diff
@@ -71,6 +88,12 @@ export const handler: AppSyncResolverHandler<
   | AdminStats
   | LaStats
   | QueryGetSchoolJoinRequestsByLaArgs
+  | QueryGetSchoolProfileArgs
+  | QueryGetSchoolsNearbyWithProfileArgs
+  | QueryGetCharitiesNearbyWithProfileArgs
+  | QueryGetCharitiesByLaArgs
+  | QueryGetCharityJoinRequestsByLaArgs
+  | QueryGetLaStatsArgs
```

Comment:
These were all unused imports which I presume need to be added here? let me know if not though